code/architecture.md

```diff
@@ -87,7 +87,7 @@ the syntax tree.
 ## Compiler
 
 Thus, the thing we call the *compiler*, which is found in the 
```

Comment:
Not sure if it’s still accurate to call this thing just the *compiler* – at least the directory name in the repository now reflects that it’s just one of several backends.

Reply:
It was this way for some years :-)

Do you have a proposal on what to write instead?